readthedocs/restapi/views/footer_views.py

```diff
@@ -71,7 +69,7 @@ def footer_html(request):
     # pylint: disable=too-many-locals
     project_slug = request.GET.get('project', None)
     version_slug = request.GET.get('version', None)
-    page_slug = request.GET.get('page', None)
+    page_slug = request.GET.get('page') or ''
```

Comment:
Is there a reason this won't work? 

```suggestion
    page_slug = request.GET.get('page', '')
```

Reply:
Because we need to pass always a string. If `page: None` is explicitly, we are going to have `None` as the final value instead of a string.

With the or, we are sure that we never have a None, but string :)

Reply:
I don't think it's possible to pass `NoneType` as a value on a GET arg. It should always be a string unless I'm not thinking of something.

Reply:
Yeah, you are right! I don't think that's possible. I was too paranoid p: